tools/cli/domainUtils.go

```diff
@@ -119,10 +115,6 @@ var (
 			Name:  FlagRetentionDaysWithAlias,
 			Usage: "Workflow execution retention in days",
 		},
-		cli.StringFlag{
```

Comment:
also please double check if this flag is currently being accessed in things like docker/scripts/buildkite etc. This is a breaking change i.e. will break scripts which already have this flag

Reply:
> also please double check if this flag is currently being accessed in things like docker/scripts/buildkite etc. This is a breaking change i.e. will break scripts which already have this flag

I have checked and resolved them in the unit tests.